infrastructure_files/getting-started-with-zitadel.sh

```diff
@@ -788,14 +834,32 @@ services:
       start_period: '20s'
 
 volumes:
-  netbird_management:
-  netbird_caddy_data:
-  netbird_crdb_data:
   netbird_crdb_certs:
-  netbird_zitadel_certs:
+EOF
+}
 
-networks:
-  netbird:
+renderDockerComposePostgres() {
+  cat <<EOF
+  # Postgres for Zitadel
+  crdb:
+    restart: 'always'
+    networks: [netbird]
+    image: 'postgres:16-alpine'
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=postgres
```

Comment:
if you want to configure a random password will be better. Maybe make it persistent using a file like the temp machine key